### PR TITLE
Add Makefile

### PR DESCRIPTION
**Pull request by rohitpaulk**

### CODE REVIEW

**Comment by f2prateek** [comment]
Awesome! Should we link up any commands to Travis CI? https://github.com/segmentio/analytics-ruby/blob/master/.travis.yml

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,17 @@
-
-test:
-	rake spec
-
-build:
-	gem build ./analytics-ruby.gemspec
-
-.PHONY: test build
+
+# Install any tools required to build this library, e.g. Node/Yarn etc.
```
Comment (f2prateek) [inline Makefile]: > e.g. Node/Yarn etc.

Could replace this with a Ruby specific example I think (e.g. Ruby, Bundler etc.)

Comment (rohitpaulk) [reply Makefile]: Fixed.

**Comment by rohitpaulk** [comment]
> Should we link up any commands to Travis CI? https://github.com/segmentio/analytics-ruby/blob/master/.travis.yml — looks like maybe make boostrap instead of the manual bundler task?

Hmm, I think that command isn't needed at all - Travis should run a bundle install by default. The default test command could be switched to `make check` though. Travis does `bundle exec rake` by default but that isn't very obvious - better to be explicit. Done in 93a0f28

**Pull request merged**